### PR TITLE
BUG: Missing writing of the nifti descrip field.

### PR DESCRIPTION
**Pull request by zivy**
The nifti descrip field is currently only read and exposed via the ITK_FileNotes metadata dictionary variable. Setting the ITK_FileNotes via the metadata dictionary has no effect while writing. This commit writes the contents of the ITK_FileNotes variable if it is found in the metadata dictionary.

### CODE REVIEW

File: Modules/IO/NIFTI/src/itkNiftiImageIO.cxx [hunk 1]
```diff
@@ -446,9 +446,9 @@ NiftiImageIO::PrintSelf(std::ostream & os, Indent indent) const
   os << indent << "NiftiImage: " << m_NiftiImage << std::endl;
   os << indent << "RescaleSlope: " << m_RescaleSlope << std::endl;
   os << indent << "RescaleIntercept: " << m_RescaleIntercept << std::endl;
-  os << indent << "ConvertRAS: " << (m_ConvertRAS ? "On" : "Off") << std::endl;
-  os << indent << "ConvertRASVectors: " << (m_ConvertRASVectors ? "On" : "Off") << std::endl;
-  os << indent << "ConvertRASDisplacementVectors: " << (m_ConvertRASDisplacementVectors ? "On" : "Off") << std::endl;
+  itkPrintSelfBooleanMacro(ConvertRAS);
```
Comment (dzenanz) [inline Modules/IO/NIFTI/src/itkNiftiImageIO.cxx:449]: This set of changes might be a separate commit.

**Review by dzenanz — APPROVED** [review]
Looks good.

File: Modules/IO/NIFTI/src/itkNiftiImageIO.cxx (hunk 1 above)
Comment (zivy) [reply Modules/IO/NIFTI/src/itkNiftiImageIO.cxx:449]: Ouch, that's not part of the changes I made. I guess I didn't look carefully when I moved things from the branch based off master to the one based off release-5.4. Will fix and force push.

**Review by dzenanz — APPROVED** [review]

**Comment by zivy** [comment]
Hi @dzenanz, please rerun the failed test if needed (it isn't related to this change) and merge when it passes. Thanks.

**Comment by dzenanz** [comment]
This will need to be rebased on top of #4811, after it gets merged. I plan to merge that as soon as its CI finished.

**Comment by blowekamp** [comment]
@dzenanz I did a quick search and didn't find it. But for patches for 5.4 what is the order of merging for master, release, releaes-5.4?

**Comment by dzenanz** [comment]
I think merge into most specific first. So first merge topic branch into `release-5.4`, then merge `release-5.4` into `release`, and finally merge `release` into `master`.

**Comment by dzenanz** [comment]
#4811 has now been merged. A plain rebase should be sufficient for a green CI.

**Comment by zivy** [comment]
Rebased and force pushed.

**Comment by dzenanz** [comment]
I merged it to `release` and `master` branches.

**Comment by zivy** [comment]
Thanks. Much appreciated.
